util-taglib/src/com/liferay/taglib/aui/InputTag.java

```diff
@@ -208,7 +208,9 @@ protected void setAttributes(HttpServletRequest request) {
 		int pos = name.indexOf(StringPool.DOUBLE_DASH);
 
 		if (pos != -1) {
-			name = name.substring(pos + 2, name.length() - 2);
+			if ((name.substring(pos + 2)).endsWith(StringPool.DOUBLE_DASH)) {
+				name = name.substring(pos + 2, name.length() - 2);
```

Comment:
I think we shouldn't because we are using some namespaces (ntlm--enabled, cas-enabled, etc) to avoid collisions between names and ids. I've checked that is affecting other inputs in portal-settings (eg loginUrl)

If this fix could cause any regression we should add custom ids for all the inputs in portal settings, something like https://github.com/jcampoy/liferay-portal/pull/36/commits/6cfd0c1560fe10b2490a263ce898b9c1d8fe7936 applied to other inputs.